src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.CsvStorageDaoImpl;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.handler.BalanceOperationHandler;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.handler.PurchaseOperationHandler;
+import core.basesyntax.handler.ReturnOperationHandler;
+import core.basesyntax.handler.SupplyOperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.report.CsvReportWriterImpl;
+import core.basesyntax.report.ReportBuilder;
+import core.basesyntax.report.ReportBuilderImpl;
+import core.basesyntax.report.ReportWriter;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.FruitStorageImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        StorageDao storageDao = new CsvStorageDaoImpl();
+        String filePath = "src/main/java/core/basesyntax/file.csv";
```

Comment:
Please place the input and output files into the `src/main/resources` folder, not in the `src/main/java` directory.